fenix/app/src/main/java/org/mozilla/fenix/search/SearchDialogFragment.kt

```diff
@@ -265,6 +266,12 @@ class SearchDialogFragment : AppCompatDialogFragment(), UserInteractionHandler {
             false
         }
 
+        if (!IncompleteRedesignToolbarFeature(requireContext().settings()).isEnabled) {
```

Comment:
Could we do this inside of the `also { }` on [L251](https://github.com/mozilla-mobile/firefox-android/pull/5903/files#diff-a2c1bafb4f78d608db40021b147b659d21280f0d7693a27f093745c1f2de3a14R251)? I think we should just call this as soon `ToolbarView` is initialized. Other than that, it's just about group all the toolbar view handling together.